Formula/o/openjdk@8.rb

```diff
@@ -123,25 +133,27 @@ def install
         --with-toolchain-type=clang
         --with-zlib=system
       ]
-
-      # Work around SDK issues with JavaVM framework.
-      if MacOS.version <= :catalina
-        sdk_path = MacOS::CLT.sdk_path(MacOS.version)
-        ENV["SDKPATH"] = ENV["SDKROOT"] = sdk_path
-        javavm_framework_path = sdk_path/"System/Library/Frameworks/JavaVM.framework/Frameworks"
-        args += %W[
-          --with-extra-cflags=-F#{javavm_framework_path}
-          --with-extra-cxxflags=-F#{javavm_framework_path}
-        ]
-        ldflags << "-F#{javavm_framework_path}"
-      # Fix "'JavaNativeFoundation/JavaNativeFoundation.h' file not found" issue on MacOS Sonoma.
-      elsif MacOS.version == :sonoma
-        javavm_framework_path = "/Library/Developer/CommandLineTools/SDKs/MacOSX13.sdk/System/Library/Frameworks"
-        args += %W[
-          --with-extra-cflags=-F#{javavm_framework_path}
-          --with-extra-cxxflags=-F#{javavm_framework_path}
-        ]
-        ldflags << "-F#{javavm_framework_path}"
+      # Work around Xcode 16 bug: https://bugs.openjdk.org/browse/JDK-8340341
+      DevelopmentTools.clang_build_version == 1600 &&
+        ENV.append_to_cflags("-mllvm -enable-constraint-elimination=0")
```

Comment:
```suggestion
      if DevelopmentTools.clang_build_version == 1600
        ENV.append_to_cflags("-mllvm -enable-constraint-elimination=0")
      end
```
or
```suggestion
      ENV.append_to_cflags("-mllvm -enable-constraint-elimination=0") if DevelopmentTools.clang_build_version == 1600
```
if we don't hit line length errors.

Reply:
That's why I've changed to what I am now, and it's a step-by-step reminder that I need to change to what I am now.